docs/application/dotnet/guides/nui/flexcontainer.md

```diff
@@ -253,14 +263,14 @@ You can modify the flex container appearance and behavior through the following
 
     **Usage:**
 
-    ```
+    ```csharp
     FlexContainer flexContainer = new FlexContainer();
     flexContainer.JustifyContent = FlexContainer.Justification.JustifySpaceBetween;
     ```
 
 -   `AlignItems`
 
-    This property specifies the alignment for flex items when they do not use all the available space on the cross axis.
+    This property specifies the alignment for flex items when they do not use all the available space on the cross axis:
```

Comment:
```suggestion     
    This property specifies the alignment for flex items when all the available space on the cross axis is not used:

Reply:
@pkalota can you please incorporate this too ?

Reply:
Sorry, I missed that. Done.